### DISCUSSION TITLE
Rageborne Strike deck

### DISCUSSION THREAD

**Discussion by jaredgeizer** [on3iro/aeons-end-randomizer#415] [Ideas]
Hello!
I’ve only been playing AE for a bit, so sorry if I say something dumb.
I really like the site so far and I’ve used it a couple different times. The turn order function is supper nice since I find shuffling the small turn order deck annoying. I was wondering if it would be possible to do something similar with the rageborne strike deck. Actually I guess it would function more like the nemesis randomizer since you shuffle the cards together every time a card is played.
I don’t know if there are any other nemesis’s that use a similar mechanic, so I don’t know how much value this would have. But keep up the great work.

P.S. dark mode would also be very cool if possible.

**Comment by on3iro** [comment]
Hi,

thanks for your feedback :)
Unfortunately building something like the strike deck is currently out of scope within the app. Not, because it might not be useful, but because I simply don't have much time I can dedicate to the project. I have a lot of family stuff going on right now and probably won't be able to do much for the app, apart from a few minor updates here and there.

That said, i might be able to build the dark theme over the next couple of weeks - can't promise for sure, though :).
